lib/DDGC/Schema/Result/User.pm

```diff
@@ -4,6 +4,8 @@ package DDGC::Schema::Result::User;
 
 use Moo;
 extends 'DDGC::Schema::Result';
+with 'DDGC::Schema::Role::Result::User::Subscription';
+
```

Comment:
@kablamo I'd be interested in your thoughts on this - pulling related stuff for a given result class out to a role.

I like it since, while I like "fat model", I dislike these multi-thousand line classes. I'm just not clear on the... idiomaticness of having highly specialised (i.e. non-reusable) roles.